thefuck/shells/generic.py

```diff
@@ -121,34 +127,109 @@ def put_to_history(self, command):
 
         """
 
+    def can_edit(self):
+        return False
+
+    def edit_command(self, command):
+        """Spawn default editor (or `vi` if not set) and edit command in a buffer"""
+        # Create a temporary file and write some default text
+        file_path = "The Fuck: Command Edit"
+        file_handle = open(file_path, "w")
+        file_handle.write(command)
+        file_handle.close()
+
+        editor = os.getenv("EDITOR")
```

Comment:
```python
editor = os.getenv("EDITOR", "vi")
```
?

Reply:
I did not know this was a feature of `getenv`